src/components/download/helperFunctions.js

```diff
@@ -268,27 +269,32 @@ export const strainTSV = (dispatch, filePrefix, nodes, nodeVisibilities) => {
     const fullAuthorInfo = getFullAuthorInfoFromNode(node);
     if (fullAuthorInfo) {
       const traitName = "author";
-      if (!headerFields.includes(traitName)) headerFields.push(traitName);
+      headerInsert(headerFields, null, traitName);
       tipTraitValues[node.name][traitName] = fullAuthorInfo.value;
       if (isPaperURLValid(fullAuthorInfo)) {
-        tipTraitValues[node.name][traitName] += `: ${fullAuthorInfo.paper_url}`;
+        headerInsert(headerFields, traitName, urlify(traitName));
+        tipTraitValues[node.name][urlify(traitName)] = fullAuthorInfo.paper_url;
       }
     }
 
     /* handle `vaccine` specially */
     const vaccine = getVaccineFromNode(node);
     if (vaccine && vaccine.selection_date) {
       const traitName = "vaccine_selection_date";
-      if (!headerFields.includes(traitName)) headerFields.push(traitName);
+      headerInsert(headerFields, null, traitName);
```

Comment:
_non-blocking nit_

There are 3 other instances of 
```js
if (!headerFields.includes(traitName)) headerFields.push(traitName);
``` 
in this file that can be replaced with the new `headerInsert` function ;)